createAccountCore/src/main/java/com/hamdan/forzenbook/createaccount/core/viewmodel/BaseCreateAccountViewModel.kt

```diff
@@ -151,7 +157,8 @@ fun BaseCreateAccountViewModel.CreateAccountState.toCreateAccountUiState(): Crea
         email = this.email,
         location = this.location,
         isDateDialogOpen = this.isDateDialogOpen,
-        isLoading = this.isLoading
+        isLoading = this.isLoading,
+        accountCreated = this.accountCreated
```

Comment:
Get in habit of trailing commas so you don't get diffs like this

Reply:
ye for sure, I'm a little inconsistent with it, but have been getting better about it though.